ignite/services/network/testutil/response.go

```diff
@@ -0,0 +1,33 @@
+package testutil
+
+import (
+	"encoding/hex"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	prototypes "github.com/gogo/protobuf/types"
+	"github.com/ignite-hq/cli/ignite/pkg/cosmosclient"
+	"google.golang.org/protobuf/runtime/protoiface"
+)
+
+func NewResponse(data protoiface.MessageV1) cosmosclient.Response {
```

Comment:
Can we add description about what this func actually do and how it helps in the unit testing?